cmd/helm/rollback.go

```diff
@@ -36,15 +37,30 @@ second is a revision (version) number. To see revision numbers, run
 `
 
 func newRollbackCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
-	client := action.NewRollback(cfg)
+	client := action.NewUpgrade(cfg)
 
 	cmd := &cobra.Command{
 		Use:   "rollback [RELEASE] [REVISION]",
 		Short: "roll back a release to a previous revision",
 		Long:  rollbackDesc,
 		Args:  require.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			_, err := client.Run(args[0])
+
+			version, err := strconv.Atoi(args[1])
```

Comment:
My one concern about removing the `rollback` action is that it makes use as a library more difficult because anyone who wants to rollback will have to duplicate this same block of added logic. If we are going to do this, we should create a helper function that grabs all the right data, merges values, and returns back the chart

Reply:
The same argument could be applied to `helm template`. There was an initiative to expose the CLI-specific logic in #5448, though that stalled out because of certain design issues with the `settings` object that made it difficult to expose as a consumable package.